openwisp-config/files/openwisp.agent

```diff
@@ -971,3 +975,15 @@ while true; do
 	# ignore SIGUSR1 signals again
 	trap "" USR1
 done
+
+# Logging function
+log_message() {
+	local level="$1"
+	local message="${2:-}"  # Default to an empty string if the second argument is not provided
```

Comment:
I would make the message mandatory. For the level I would fall back to `daemon.info` if none is provided.